### PR TITLE
Add grantee FAQ

### PR DESCRIPTION
**Pull request by dergigi**
Build preview:

- [/faq/grantee](https://os-website-git-grantee-faq-open-sats.vercel.app/faq/grantee)

### CODE REVIEW

**Review by Arvin21M — APPROVED** [review]
This looks great! Build preview looks fine and all the links in the FAQ list move down to the proper place on the page. Some feedback below on possible improvements:

1. Considering the list of FAQ's are robust, it might be easier to track from a readers perspective if the bullet points were numbered.
2. Under "What does an ideal progress report look like?"
-- It would be helpful to include a couple of redacted examples, if not add some language to convey that we appreciate clarity and conciseness in their reporting.

Aside form the notes above, this looks ready to go 🚀

File: data/pages/faq-grantees.mdx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+We select people who are aligned with our mission and values, and we trust that
+the rest will fall into place. Our expectation is that you're acting in good
+faith.
+
+### Does my grant renew automatically?
+
+No, each grant has a set end date listed in the grant agreement. If you would
+like to apply for another grant, consider applying 60-90 days before the end of
+your current grant in order to give us enough time to evaluate and process your
+application.
```
Comment (0xB10C) [inline data/pages/faq-grantees.mdx:185]: Might be worthwhile to clarify that this is not the case for LTS grants (or if it is, then that).

Comment (BoltTouring) [reply data/pages/faq-grantees.mdx:185]: LTS grants are two years, but this applies to LTS grants as well. Applying 60-90 days before the end of the grant is ideal for all grantees that would like to apply for further funding.

File: data/pages/faq-grantees.mdx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+### How does OpenSats evaluate the work done by a grantee?
+
+Every grantee must submit a progress report every 90 days. A markdown template
+is provided when we request your first report.
+
+Reports are evaluated by:
+
+- A technical committee
+- One or multiple board members
+- At least one person from our operations team
```
[the remark this answers is not in the document]
Comment (BoltTouring) [reply data/pages/faq-grantees.mdx:101]: It depends on the grant. I like the way it's currently worded.

**Comment by BoltTouring** [comment]
>        -- It would be helpful to include a couple of redacted examples, if not add some language to convey that we appreciate clarity and conciseness in their reporting.

We will put up some example reports in the near future - I think it's best to keep it separate from the FAQ.

File: data/pages/faq-grantees.mdx (hunk 1 above)
Comment (lucasmogu) [reply data/pages/faq-grantees.mdx:185]: @0xB10C initially a few LTS grants (yours included) were approved for 12 months. A few months later, the board decided to default all existing and new LTS grants to 24 months. So any LTS grant that had been approved for only 12 months (yours included) will be auto-renewed for an additional 12-month period. At the end of the 24-month LTS cycle, the Q&A above will apply equally to all LTS grants.

File: data/pages/faq-grantees.mdx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+At OpenSats, we allocate funds from our generous donors to developers and
+educators working on free and open-source projects in the Bitcoin and Nostr
+ecosystems. Supporting our grantees is foundational to what we do. The OpenSats
+[operations team](/about#operations) is available to address
+any concerns, questions, and unforeseen circumstances that may arise. We have an
+open communication channel via
+[support@opensats.org](mailto:support@opensats.org). Please use this email for
+any inquiries.
+
+Don’t hesitate to contact us any time. Whenever possible, please let us know in
```
[the remark this answers is not in the document]
Comment (lucasmogu) [reply data/pages/faq-grantees.mdx]: changed to "anytime"

**Pull request merged**